stable/pipelines/templates/ml-pipeline/apiserver/deployment.yaml

```diff
@@ -21,6 +21,8 @@ spec:
       containers:
       - name: ml-pipeline-api-server
         env:
+#        - name: OBJECTSTORECONFIG_SECURE
+#          value: "false"
```

Comment:
🤔 this seems important (I dont remember what it does though)

Reply:
Yeah I saw it was added but didn't want to add it - cos secure = false... WDYT?

Reply:
question is if we need it or not, not how it looks in the chart 🤷